### DISCUSSION TITLE
Darker github action fails @1.7.1

### DISCUSSION THREAD

**Discussion by mazen-mohamed-tamhub** [akaihola/darker#485] [Q&A] [answered]
Hi,

My Github workflow fails suddenly today with the following message, any ideas?

![image](https://user-images.githubusercontent.com/127022007/227785717-c2cbb177-c3a6-447f-9224-c091931252e9.png)

**Comment by MatthijsBurgh** [comment]
I also experience this, see https://github.com/MatthijsBurgh/epydoc/actions/runs/4530657654/jobs/7979867795?pr=11

So https://github.com/akaihola/darker/commit/639ccb7e54ef44d0a0e272c8602f7176c0570cbc is the bad guy here. Which probably worked for you in the same repository, but not outside of it.

I suggest to revert this change. I don't see the benefit of doing a checkout here of the darker repo. Which could even result in permission issues. When people start using different tokens. (I am not sure about it, but it is much more risk involved than reverting this change.)

**Reply by akaihola** [reply, marked as answer]
Reverted in #486.
